input/input.c

```diff
@@ -43,9 +43,9 @@
 #define CODE_FAKE_NOT_CHARGING 10021
 
 #define NUM_FDS 4U
-int    nfds                  = 0;
+int    nfds                  = 0;  // for select()
 int    inputfds[NUM_FDS]     = { -1, -1, -1, -1 };
```

Comment:
The NUM_FDS is not necessary I believe, it can be retrieved from ARRAY_SIZE(inputfds).